hax/hax.ini

```diff
@@ -135,6 +135,12 @@ corrections_definitions = {
         {"run_min": 10223, "run_max": 12089, "correction": "FDC_SR1_data_driven_time_dependent_3d_correction_part3_v1.json.gz"},
         {"run_min": 12090, "correction": "FDC_SR1_data_driven_time_dependent_3d_correction_part4_v1.json.gz"}
     ],
+    "fdc_3d_tfnn": [
```

Comment:
@jingqiangye Can you add SR0 as well? (Otherwise, I think this will crash on SR0 files.)

@/all When pushing code/analyses from now, for completeness, we should ensure SR0+SR1 are both included.

Reply:
Fixed in https://github.com/XENON1T/hax/pull/174/commits/f60ce1079698a87dfb4cba569438ba73f9a961f8